apps/cloudfront-functions/src/__tests__/viewer-request-handler.test.ts

```diff
@@ -1,5 +1,9 @@
 import { describe, it, expect } from 'vitest';
-import { handler } from '../viewer-request-handler';
+import rewire from 'rewire';
+
+const rewired = rewire('./dist/viewer-request-handler');
```

Comment:
```suggestion

// the path is not ../../dist/<file> because the resolution starts from the
// 'cloudfront-functions' folder
const rewired = rewire('./dist/viewer-request-handler');
```